config/mainnet.go

```diff
@@ -77,6 +78,14 @@ func MainnetConfig() Config {
 
 	hare4conf := hare4.DefaultConfig()
 	hare4conf.Enable = false
+
+	oldAtxSyncCfg := sync2.DefaultConfig()
+	oldAtxSyncCfg.MultiPeerReconcilerConfig.SyncInterval = time.Hour
+	oldAtxSyncCfg.MaxDepth = 16
+	newAtxSyncCfg := sync2.DefaultConfig()
+	newAtxSyncCfg.MaxDepth = 21
+	newAtxSyncCfg.MultiPeerReconcilerConfig.SyncInterval = 5 * time.Minute
```

Comment:
I don't like how this config is structured, what is the meaning of old AtxSyncCfg and NewAtxSyncCfg? Why are they part of `"main"` -> `"V2"`? What is `"V2"` in this context? Shouldn't they be part of the `"main"` -> `"AtxSync"` config object?

Reply:
EDIT: fixed spelling

Reply:
This might cause confusion as there's currently V1 sync for ATXs, too, which is also configured in the syncer config.
When V2 is used in server-only mode, both V1 and V2 syncs are used at the same time.

Reply:
The idea is to use reasonable defaults so that most users will not have to adjust sync settings.
Then at some time in the future we might want to restructure the config to remove all the v1 stuff and keep v2 sync only.

Reply:
old vs new in this context means old epochs vs the current epoch

Reply:
Ah so it is actually `PrevEpochSyncConfig` and `CurrentEpochSyncConfig`? 🙂

Reply:
> The idea is to use reasonable defaults so that most users will not have to adjust sync settings. Then at some time in the future we might want to restructure the config to remove all the v1 stuff and keep v2 sync only, moving it out of this `v2` field.

The reasonable defaults are already specified as part of the `syncer.DefaultConfig`, my point was that we don't need to overwrite this values again with the same values in `config.MainnetConfig` 🙂

Reply:
> This might cause confusion as there's currently V1 sync for ATXs, too, which is also configured in the syncer config. When V2 is used in server-only mode, both V1 and V2 syncs are used at the same time.

But this is a technical detail that I think should not be exposed via the config, or at least in a different way. Maybe instead of `V2` we can call it something like "reconciliationSync" or similar. Especially since "V2" has negative connotations: https://en.wikipedia.org/wiki/V-2_rocket

Reply:
Actually there's a bit of confusion re current vs prev, and I was imprecise in my above comment.
When a new epoch starts, activations don't start to be published in that epoch immediately, and consequently, the more efficient (but more memory and CPU hungry per missing/extra set element) sync is to be applied to the new epoch some time later (`EpochEndFraction` in the existing sync config). That's why the "new" epoch is not necessarily current one, and the "old" epoch is not necessarily the previous. We could probably come up with better naming but I'm unsure `current` / `prev` is the right choice here.

Otherwise agree, will fix

Reply:
Renamed to `v2` part to `reconc-sync` (`ReconcSync`).